users/vieting/experiments/switchboard/hybrid/feat/baseline_args.py

```diff
@@ -239,13 +244,22 @@ def get_returnn_config(
                     network["source"] = specaug_layer_only_time(in_layer=["features"])
                     prolog = get_funcs_only_time()
                 else:
-                    network["source"] = specaug_layer_jingjing(in_layer=["features"])
-                    prolog = get_funcs_jingjing()
+                    if specaug_shuffled:
```

Comment:
`specaug_shuffled` is only checked here in this nested `if`-`else` structure. You might add assertions above that `specaug_suffled` is `False`, only to make sure that there are no confusions.

Reply:
Done